packages/framework/src/Component/Enum/AbstractEnumCasesProvider.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Enum;
+
+use Shopsys\FrameworkBundle\Component\Reflection\ReflectionHelper;
+
+class AbstractEnumCasesProvider
```

Comment:
I would rename this to something more generic e.g. `AbstractEnum`, as it can be used for more uses

Reply:
renamed :+1: